OpenRA.Utility/Program.cs

```diff
@@ -43,9 +43,16 @@ static void Main(string[] args)
 			Log.AddChannel("perf", null);
 			Log.AddChannel("debug", null);
 
-			Game.InitializeSettings(Arguments.Empty);
+			var arguments = new Arguments(args);
+			args = arguments.GetSkippedValues();
```

Comment:
Another option would be to read `Engine.ModSearchPaths` from an environment variable, but we will need to think a bit about how we want to handle the other launcher -> game/utility/server bits first if we want to do this.